t/mojolicious/app.t

```diff
@@ -419,6 +419,11 @@ $t->get_ok('/../../mojolicious/secret.txt')->status_is(404)
 $t->get_ok('/.../mojolicious/secret.txt')->status_is(404)
   ->header_is(Server => 'Mojolicious (Perl)')->content_like(qr/Page not found/);
 
+# Try to access a file which is not under the web root via path
+# traversal (url_escaped backslashes)
```

Comment:
`url_escaped` looks weird, it's neither a real word not a function.

Reply:
what's better `url escaped` or just `escaped`?